### DISCUSSION TITLE
Add a `noir.destroy()` method to `noir_js`

### DISCUSSION THREAD

**Discussion by signorecello** [noir-lang/noir#2944] [Ideas]
### Problem

We're allocating resources everytime we call `noir.init()`. These will build up specially if state isn't managed correctly in the frontend.

### Happy Case

Add a noir.destroy() to shut down the backend to avoid using too many resources. This essentially will just call the upstream `backend.destroy()` method, which in the case of `bb.js` it's exactly how it's called

### Would you like to submit a PR for this Issue?

No

**Comment by Savio-Sou** [comment]
Closed with https://github.com/noir-lang/noir/pull/3105.
